kong/llm/drivers/bedrock.lua

```diff
@@ -322,25 +322,52 @@ function _M.subrequest(body, conf, http_opts, return_res_table)
   end
 
   -- may be overridden
-  local url = (conf.model.options and conf.model.options.upstream_url)
-    or fmt(
-    "%s%s",
-    ai_shared.upstream_url_format[DRIVER_NAME],
-    ai_shared.operation_map[DRIVER_NAME][conf.route_type].path
-  )
+  local f_url = conf.model.options and conf.model.options.upstream_url
+
+  if not f_url then  -- upstream_url override is not set
+    local uri = fmt(ai_shared.upstream_url_format[DRIVER_NAME], identity_interface.interface.config.region)
+    local path = fmt(
+      ai_shared.operation_map[DRIVER_NAME][conf.route_type].path,
```

Comment:
is the usage of `fmt` correct here - ie, is `path` actually a format string? would it be a better approach to call `fmt` only once instead, below?

Reply:
Yes they are both format strings, different ones.

This was the tidiest way to do it without nesting loads of `fmt` and it looking weird.

But I've reduced it down to only the two now, instead of 4.